server/models/customer.js

```diff
@@ -0,0 +1,46 @@
+//customer: jobs: name, description, required skills, 
+//need date, city, job by id, proposals by id
+
+//finds base in index.js
+const Base = require('./index');
+
+const Customer = Base.descriminator('Customer', new mongoose.Schema({
```

Comment:
I think we can create a jobSchema as its own file and then use that in the Customer model. The Customer model would have a 'jobs' field which is an array of 'jobSchema' objects